src/components/CodeView/index.tsx

```diff
@@ -45,6 +47,9 @@ const renderCode = ({
   if (shouldHighlight) {
     const ast = refractor.highlight(code, language);
     value = ast.length === 0 ? code : ast.map(mapWithDepth(0));
+  } else {
+    // If the code cannot be highlighted, truncate the line as well.
+    value = code.substring(0, HIGHLIGHT_WIDE_LINE_LENGTH);
```

Comment:
I was thinking about this last night, and I wonder if this should be:

```js
value = code.length > HIGHLIGHT_WIDE_LINE_LENGTH ? code.substring(0, HIGHLIGHT_WIDE_LINE_LENGTH) : code;
```

I'm not sure how efficient js is at doing a `substring`, so maybe we shouldn't be calling `substring` for every line, but only for those that are in fact too wide.

WDYT?

Reply:
I think that's a micro-optimization. Let's keep things as simple as possible, I don't think this particular change will lead to a significant improvement.

(From what I am seeing, even syntax highlighting is not always the problem)